### PR TITLE
pw-cat, pw-record: remove deprecated example

### PR DESCRIPTION
**Pull request by CheesyChocolate**
Option `--list-targets` is no longer available in the recent versions of `pw-record`
[arch wiki page](https://wiki.archlinux.org/title/PipeWire#Microphone_is_not_detected_by_PipeWire)
[a related issue in pipewire git repo](https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/2564)

- [x] The page(s) are in the correct platform directories: common, linux, osx, windows, sunos, android, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](https://github.com/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

- **Version of the command being documented (if known):**
pipewire-audio 1:0.3.83-1
Compiled with libpipewire 0.3.83

### CODE REVIEW

**Comment by CheesyChocolate** [comment]
should we add more example if 3 example is not enough? (after removing deprecated example, only 3 remains)

**Comment by Managor** [comment]
If you can think of valid use cases. There's no need to forcibly list more commands than are needed.

File: pages/linux/pw-cat.md [hunk 1]
```diff
@@ -3,18 +3,10 @@
 > Pipewire tool for playing and recording audio files.
 > More information: <https://fedoraproject.org/wiki/QA:Testcase_PipeWire_PipeWire_CLI>.
```
Comment (Managor) [inline pages/linux/pw-cat.md:4]:
```suggestion
> Play and record audio files through pipewire.
> More information: <https://fedoraproject.org/wiki/QA:Testcase_PipeWire_PipeWire_CLI>.
```

File: pages/linux/pw-record.md [hunk 2]
```diff
@@ -3,10 +3,6 @@
 > Shorthand tool for pw-cat --playback.
 > More information: <https://fedoraproject.org/wiki/QA:Testcase_PipeWire_PipeWire_CLI>.
```
Comment (Managor) [inline pages/linux/pw-record.md:4]:
```suggestion
> Record audio files through pipewire
> Shorthand for pw-cat --record.
> More information: <https://fedoraproject.org/wiki/QA:Testcase_PipeWire_PipeWire_CLI>.
```

**Review by Managor — CHANGES_REQUESTED** [review]
Lets fix some of my own mistakes from when I wrote this page to conform with the style guide.

**Comment by CheesyChocolate** [comment]
I will update turkish tldr pages as soon as I get a reply from a turkish friend. (his name is not google translator!!!)

**Review by Managor — APPROVED** [review]

**Pull request merged**